src/middlewared/middlewared/plugins/iscsi_/extents.py

```diff
@@ -198,8 +198,8 @@ async def do_delete(self, id, remove, force):
             await self._service_change('iscsitarget', 'reload')
 
     @private
-    async def validate(self, data):
-        data['serial'] = await self.extent_serial(data['serial'])
+    def validate(self, data):
```

Comment:
If you change the function definition from `async` to non-async, you need to also change all its invocations from `await` to conventional calls